tools/defaults.bzl

```diff
@@ -17,6 +18,29 @@ _DEFAULT_TSCONFIG_TEST = "//src:tsconfig-test"
 # Re-exports to simplify build file load statements
 markdown_to_html = _markdown_to_html
 
+def _compute_module_name(testonly):
+    current_pkg = native.package_name()
+
+    # For test-only targets we do not compute any module name as
+    # those are not publicly exposed through the `@angular` scope.
+    if testonly:
+        return None
+
+    # We generate no module name for files outside of `src/<pkg>` (usually tools).
+    if not current_pkg.startswith("src/"):
+        return None
+
+    # Skip module name generation for Bazel packages which are not built as NPM package
+    # or not scoped under `@angular/`.
+    if current_pkg.startswith("src/universal-app") or \
```

Comment:
Maybe just ends with "-app"?

Reply:
I updated to check if the package name contains "-app"